recipes/sctk/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "sctk" %}
+{% set version = "2.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 43bd2bfe60e0ad04a563ddef025e84fca4c073f8f7c02f056b59fb8e2ec4f022
+
+build:
+  number: 3
```

First published package of the same version should start as build 0.

```suggestion
  number: 0
```